server/api/core/core.js

```diff
@@ -2,7 +2,7 @@ import url from "url";
 import { merge, uniqWith } from "lodash";
 import { Meteor } from "meteor/meteor";
 import { EJSON } from "meteor/ejson";
-import { Jobs, Packages, Shops } from "/lib/collections";
+import {Jobs, Packages, Shops, Accounts} from "/lib/collections";
```

Comment:
you may want to be consistent with leaving whitespace after and before the opening and closing curly braces respectively, just like in the other `import` statements on line 2 - 4